jina/serve/executors/__init__.py

```diff
@@ -110,6 +112,16 @@ def __init__(
         self._add_runtime_args(runtime_args)
         self._init_monitoring()
         self.logger = JinaLogger(self.__class__.__name__)
+        if __dry_run_endpoint__ not in self.requests:
+            self.requests[__dry_run_endpoint__] = self._dry_run_func
+        else:
+            self.logger.warning(
+                f' Endpoint {__dry_run_endpoint__} is defined by the Executor. Be aware that this endpoint is usually reserved to enable health checks from the Client through the gateway.'
+                f' So it is recommended not to expose this endpoint. '
+            )
+
+    def _dry_run_func(self, *args, **kwargs):
+        pass
```

Comment:
Should not we log something here (in debug level )  ?

Reply:
in theory u will see `DataRequest on `__dry_run_endpoint__` received

Reply:
yes you are right